windowsdefenderatp_connector.py

```diff
@@ -783,9 +783,9 @@ def _handle_test_connectivity(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
         self.save_progress(DEFENDERATP_MAKING_CONNECTIVITY_MSG)
 
-        self.save_progress(f"Login URL: {self._login_url}")
-        self.save_progress(f"Graph URL: {self._graph_url}")
-        self.save_progress(f"Resource URL: {self._resource_url}")
+        self.save_progress("Login URL: {}".format(self._login_url))
```

Comment:
Normally, this should be valid syntax from Python3.6 onwards.

Reply:
Agreed!
But also wanted to keep things consistent

Reply:
Understood - that makes sense, consistency is good :) Thanks!